Alloy/builtins/moment/lang/en-sg.js

```diff
@@ -1,6 +1,4 @@
 //! moment.js locale configuration
-//! locale : English (Singapore) [en-sg]
```

Comment:
This file keeps appearing and reappearing for me, will see what is actually correct